beluga/util.py

```diff
@@ -0,0 +1,9 @@
+from shapely import wkb
+
+
+def wkt_to_location(wkt_obj):
```

Comment:
I think this file should go in the tests folder since we only use it there (and I don't think the actual application will need to go from db -> lat/lon, just lat/lon -> db)